ooniprobe/Model/Database/Result.mm

```diff
@@ -111,6 +111,14 @@ -(NSString*)getLogFile:(NSString*)test_name{
     return [NSString stringWithFormat:@"%@-%@.log", self.Id, test_name];
 }
 
+-(NSString*)getLocalizedStartTime{
+    //TODO we might need a copy for this
+    if (self.start_time == nil)
+        return NSLocalizedString(@"TestResults.UnknownASN", nil);
+    NSString *localizedDateTime = [NSDateFormatter localizedStringFromDate:self.start_time dateStyle:NSDateFormatterShortStyle timeStyle:NSDateFormatterShortStyle];
+    return localizedDateTime;
```

Comment:
This seems to be the same code as the above. Can we refactor this into a common function instead of it being a class method?